### ISSUE TITLE
p:message with display="tooltip" is not readable with Aristo in IE11

### ISSUE THREAD

**Issue by NicolaIsotta** [primefaces/primefaces#2806]
## 1) Environment
- PrimeFaces version: showcase (6.2-SNAPSHOT)
- Affected browsers: IE11

## 2) Expected behavior
Tooltip error message should be readable

## 3) Actual behavior
On IE 11:
![tooltip ie11](https://user-images.githubusercontent.com/18428155/31492140-08cd6528-af4a-11e7-8005-382e351be557.jpg)
Also, there may be some css issues also with other browsers (ex. Edge):
![tooltip edge](https://user-images.githubusercontent.com/18428155/31492214-6314a56e-af4a-11e7-816d-3e8b33310ff2.jpg)

## 4) Steps to reproduce
Open showcase, choose Aristo theme, go to Messages -> Messages ( https://www.primefaces.org/showcase/ui/message/messages.xhtml ), click on submit.

**Comment by tandraschko** [comment]
Does it work with other themes?

**Comment by NicolaIsotta** [comment]
With Omega it works. I haven't tested community themes.
I dicovered the cause with a little bit of F12.
The `p:tooltip` content has both the `ui-state-error` and the `ui-tooltip-text` classes.

`ui-state-error` sets both `background-color` and `background-image` with the `background` shorthand.

https://github.com/primefaces/primefaces/blob/710af7e68da8d47f60185d2c59cdbc4c854f102c/src/main/resources/META-INF/resources/primefaces-aristo/theme.css#L168-L176

`ui-tooltip-text` resets only the `background-color`
https://github.com/primefaces/primefaces/blob/413c3f1435051cb969dc1f007e93c1de4d885277/src/main/resources/META-INF/resources/primefaces/tooltip/tooltip.css#L17-L21

The fix I've found is easy: adding `background-image: none;` or replacing `background-color: rgb(76, 76, 76);` with `background: rgb(76, 76, 76);` in the `.ui-tooltip .ui-tooltip-text` rule solves the problem for all the affected themes, and there should be no drawbacks.
